plugins/plugin-chart-echarts/src/Pie/transformProps.ts

```diff
@@ -102,6 +102,7 @@ export default function transformProps(chartProps: EchartsPieChartProps): PieCha
     emitFilter,
   }: EchartsPieFormData = { ...DEFAULT_LEGEND_FORM_DATA, ...DEFAULT_PIE_FORM_DATA, ...formData };
   const metricLabel = getMetricLabel(metric);
+  const minShowLabelAngle = (showLabelsThreshold || 0) * 3.6;
```

Comment:
It turns out the `TextControl` with `isFloat: true` returns an empty string when unset, making it important to replace a falsy value for `showLabelsThreshold` with zero.